### DISCUSSION TITLE
Day of the celebration

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1683] [Ideas] [answered]
### Is your idea related to a problem?

No

### Describe your idea

On the day of the celebration, the M3 presents an error because there is no schedule for the midweek meeting, I know that it is possible to change the day of the meeting and add manual media, but would it be possible to automate it? Would M3 already know that this year it will be April 4th and the meeting will automatically open on Tuesday? The suggestion is for M3 to have the options to insert exertions, as we already have in the superintendent's visit. When entering the date of the celebration, M3 will already understand that there will be no meeting in the middle of the week or on the weekend (depending on the date) and the program will not look for media to download.

### Describe alternatives you've considered

An error is displayed, but when clicking to close, it is possible to add the media of the day.

**Comment by mtdvlpr** [comment, marked as answer]
@e-Feitosa, I have tackled the error and in the next release there won't be one for the memorial week.
